autogen/agentchat/contrib/gpt_assistant_agent.py

```diff
@@ -173,10 +181,60 @@ def _get_run_response(self, thread, run):
         """
         while True:
             run = self._wait_for_run(run.id, thread.id)
-            if run.status == "completed":
+            if run.status == "failed":
+                new_messages = []
+                logger.info(f'Run: {run.id} Thread: {thread.id}: failed...')
+                if run.last_error:
```

Comment:
Failed, but will there be no last error? What I'm thinking is, can we reduce the number of messages

Reply:
OK, openai need to improve it